### PR TITLE
Pouta check

### PR DESCRIPTION
**Pull request by UFOPOLI**
The document "getting started" was only available as a ~ file. Is it also supposed to be included?

### CODE REVIEW

File: docs/cloud/pouta/command-line-tools.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+This article describes using Pouta on the command line. If not done
+already, start by [installing the OpenStack tools](install-client.md).
+
+You can do most of the things you need in the OpenStack (Horizon)
+web interface of cPouta and ePouta. However, sometimes it is more practical,
+or even necessary, to use command line tools. OpenStack is divided into
+subcomponents that each take care of specific tasks such as managing
+virtual machines, volumes or images. Each of these subcomponents has
+its own API which can be accessed either programmatically or in a
+terminal using the command line tools that are introduced in this
```
[the remark this answers is not in the document]
Comment (UFOPOLI) [reply docs/cloud/pouta/command-line-tools.md]: Sure, except that "subcomponents" is one word, and you probably meant "chapter"?

File: docs/cloud/pouta/command-line-tools.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
 chapter.
 
 The command line tools are used to control your use of the service, so
-they  should  be installed  either  on  your local  laptop/desktop  or
+they should be installed either on your local laptop/desktop or
 another server which you will use to manage the service.
 
-# Using Pouta from the command line
+# Using Pouta on the command line
```
[the remark this answers is not in the document]
Comment (UFOPOLI) [reply docs/cloud/pouta/command-line-tools.md]: "From" in this kind of cases sounds like a fennicism to me but if you think it's technically more descriptive, why not.

File: docs/cloud/pouta/command-line-tools.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (oscarkraemer) [reply docs/cloud/pouta/command-line-tools.md]: I did not mean chapter. I think the whole section is pretty bad as it is. The concept of subcomponents is pretty bad word and is not used anywhere else in this documentation. I suggest that the section would be removed/changed to: ``` You can do most of the things you need in the OpenStack web interface (Horizon). However, sometimes it's handier, or even necessary to use command line tools. In this chapter we are discussing the command line tools used in Pouta```

[the remark this answers is not in the document]
Comment (UFOPOLI) [reply docs/cloud/pouta/command-line-tools.md]: `However, sometimes it is handier, or even necessary, to use command line tools. In this chapter, we discuss...`

**Comment by Nortamo** [comment]
There were two conflicts which were resolved in the following way:
### 1 
In `docs/cloud/pouta/known-problems.md` the pouta FAQ entries link was incorporated from master.

### 2 
In `cloud/pouta/images.md `  , removed the master branch version of the CentOS-7  chapter.

File: docs/cloud/pouta/storage.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
 
-This article collects the different storage options available in CSC
-cloud environment. In order to utilize the storage options, you'll need
-to have an account, a project and a virtual machine.
+This article introduces the storage options available in CSC's
+cloud environment. Using the storage requires
+an account, a project, and a virtual machine.
 
-There  is at  least one  type of  storage available  on every  virtual
+There is at least one type of storage available on every  irtual
```
Comment (Aegil) [inline docs/cloud/pouta/storage.md]:
```suggestion
There is at least one type of storage available on every  virtual
```

File: docs/cloud/pouta/additional-services.md [hunk 4]
```diff
@@ -1,66 +1,69 @@
...
-    it  should be  mapped to.  You do  not need  to send  mail if  the
-    floating IP  is moved to  another machine,  as long as  the domain
-    info for the floating IP still is correct.
--     When  you   no   longer  need   the   mapping,  please   contact
-    cloud-support@csc.fi so we can remove the reverse DNS entry.
-
-We reserve  the right to  clean up old  reverse DNS records  where the
-forward DNS  records do  not match  anymore. We  will also  remove the
+- Point the DNS name to the server you want.
+- Send a mail to cloud-support@csc.fi with the IP/hostname pair
```
Comment (Aegil) [inline docs/cloud/pouta/additional-services.md]:
```suggestion
- Send a mail to servicedesk@csc.fi with the IP/hostname pair
```
Our new standard is all mail goes via servicedesk

File: docs/cloud/pouta/additional-services.md [hunk 5]
```diff
@@ -1,66 +1,69 @@
...
+ you want to be mapped. Also send the UUID on the virtual machine
+ it should be mapped to. You do not need to send a mail if the
+ floating IP is moved to another machine, as long as the domain
+ info for the floating IP still is correct.
+- When you no longer need the mapping, please contact
+ cloud-support@csc.fi so we can remove the reverse DNS entry.
```
Comment (Aegil) [inline docs/cloud/pouta/additional-services.md]:
```suggestion
 servicedesk@csc.fi so we can remove the reverse DNS entry.
```

File: docs/cloud/pouta/command-line-tools.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Aegil) [reply docs/cloud/pouta/command-line-tools.md]: Using "from the" sounds more natural to me also. 
Or then Using Pouta command line tools.

File: docs/cloud/pouta/command-line-tools.md [hunk 6]
```diff
@@ -33,12 +33,12 @@ that uses the common openstack command line tool.
...
 
-##### Openstack Commands and help
+##### Openstack commands and help
 
     openstack -h
 
-You can get a list of  available openstack subcommands by appending -h
-after openstack command. If you wish to see the options for a specific
+You can get a list of the available OpenStack subcommands by appending -h
+after the OpenStack command. If you wish to see the options for a specific
```
Comment (Aegil) [inline docs/cloud/pouta/command-line-tools.md]: The second mention is the cli tool : openstack. The first is the project name, so second should be lower case or then highlighted.

**Pull request merged**